definitions/tools/gather_to_sub_directory_dirs.cwl

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: ["/bin/bash","directory_gatherer.sh"]
+
+requirements:
+    - class: DockerRequirement
+      dockerPull: "ubuntu:xenial"
```

Comment:
Is it time to start basing things on `bionic` or even `focal`?

Reply:
I changed to the `focal` image